source/core/slang-stream.h

```diff
@@ -138,7 +138,10 @@ class OwnedMemoryStream : public MemoryStreamBase
     void setContent(const void* contents, size_t contentsSize)
     {
         m_ownedContents.setCount(contentsSize);
-        ::memcpy(m_ownedContents.getBuffer(), contents, contentsSize);
+        if (contentsSize > 0)
+        {
+            ::memcpy(m_ownedContents.getBuffer(), contents, contentsSize);
+        }
```

Comment:
This change is unnecessary.
`memcpy` can handle when the size is zero.
However, the behavior is undefined if `contents` is nullptr.

Reply:
I don't want us hit the undefined behavior, so made this change.

Reply:
If I wasn't clear, I was suggesting the following code,
```
        if (contents)
        {
            ::memcpy(m_ownedContents.getBuffer(), contents, contentsSize);
        }
```
Your current code can still hit the undefined behavior.
If contents is nullptr, the behavior is undefined.